.gitignore

```diff
@@ -1,3 +1,5 @@
+## Jekyll build files
```

Comment:
It is not just Jekyll, it is also artifacts from mvn.

maybe split it up, if you want to mark it as such.